### DISCUSSION TITLE
Instability? serverless functions

### DISCUSSION THREAD

**Discussion by tihmdcc** [vercel/community#3867] [Help]
### Summary

Hey guys!
Today (31-08-23) in the afternoon when trying to use serverless functions on our website we came across the following error:

"This Serverless Function has timed out.

Your connection is working correctly.

Vercel is working correctly.

504: GATEWAY_TIMEOUT
Code: FUNCTION_INVOCATION_TIMEOUT
ID: gru1::khcsz- -"

In the morning the services were working normally.
We tried changing the region to 'sfo1' to test if it would work again, but it still didn't work. [In the error code, even changing the region code, it also remained as gru1]

Thank you in advance for your attention!

### Steps to Reproduce

Run some page that requires a serverless functions

**Comment by amyegan** [comment]
Hey @tihmdcc. A [504: FUNCTION_INVOCATION_TIMEOUT](https://vercel.com/docs/errors/application-error-codes#504:-function_invocation_timeout) error means the Serverless Function reached the [timeout threshold](https://vercel.com/docs/limits/overview#serverless-function-execution-timeout). If you can share more about what the function does, I might be able to help you figure out what's causing it to run for too long.

**Comment by tihmdcc** [comment]
The problem has been resolved.
It looks like they blocked our serveless functions from running. We upgraded to a pro account and it was resolved.
Thanks!
